avogadro/core/neighborperceiver.h

```diff
@@ -29,21 +29,29 @@ class AVOGADROCORE_EXPORT NeighborPerceiver
    *
    * @param points Positions in 3D space to detect neighbors among.
    * @param maxDistance All neighbors strictly within this distance will be detected.
-   *                    Between 1 and 2*sqrt(3) times this distance, some could be included.
-   *                    Beyond 2*sqrt(3) times this distance, none will be included.
    *                    Should be as low as possible for best performance.
    */
   NeighborPerceiver(const Array<Vector3> points, float maxDistance);
   
   /**
    * Returns a list of neighboring points. Linear time to number of neighbors.
+   * Can return some neighbors up to 2*sqrt(3) times the maximum distance.
    *
    * @param point Position to return neighbors of, can be located anywhere.
    */
-  const Array<Index> getNeighbors(const Vector3 point) const;
+  const Array<Index> getNeighborsInclusive(const Vector3 &point) const;
```

Comment:
I guess I'm a little unclear on the reasoning for the near-duplicate method. I guess the point is to minimize passing references / copies?

If so, it might be useful to mention in the top method that it can allocate extra copies and in-place is preferred if possible.

Reply:
Done!